src/frontends/llvm/l2s_instr.c

```diff
@@ -394,11 +394,11 @@ EmittedInstr convert_instruction(Parser* p, Node* fn_or_bb, BodyBuilder* b, LLVM
                     String name = get_string_literal(target->arena, name_node);
                     assert(name);
                     set_variable_name((Node*) target, name);
-                    return (EmittedInstr) {};
+                    //return {};
```

Comment:
TODO: do not leave it without return, this thing requires attention and msvc doesn't like the previous statement

Reply:
fixed in https://github.com/Hugobros3/shady/commit/14161c6f96cb3cfb86bc430d0e5a813b98d0b362